superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -539,9 +539,13 @@ class SqlEditor extends React.PureComponent {
   }
 
   renderQueryLimit() {
-    const menuDropdown = (
+    // Adding SQL_MAX_ROW value to dropdown
+    const { maxRow } = this.props;
+    LIMIT_DROPDOWN.push(maxRow);
+
+    return (
       <AntdMenu>
-        {LIMIT_DROPDOWN.map(limit => (
+        {[...new Set(LIMIT_DROPDOWN)].map(limit => (
```

Comment:
I'm not familiar with `Set` in Javascript, but in Python they're not ordered. Will this affect the order of the elements?

Reply:
Thanks, @rusackas!